test/support_stdlib/oneapi_std_test_config.h

```diff
@@ -0,0 +1,33 @@
+// -*- C++ -*-
+//===-- oneapi_std_test_config.h ------------------------------------------------===//
+//
+// Copyright (C) 2017-2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEAPI_STD_TEST_config_H
+#define _ONEAPI_STD_TEST_config_H
+
+#include "../support/test_config.h"
+#include "../support/utils.h"
+
+#define _ONEAPI_STD_TEST_STRING(X) _ONEAPI_STD_TEST_STRING_AUX(oneapi/dpl/X)
```

Comment:
Should this macro be moved after the definition of `_ONEAPI_STD_TEST_STRING_AUX` on the following line?

Reply:
Fixed,